### DISCUSSION TITLE
Physical units (mm, cm, in, etc)

### DISCUSSION THREAD

**Discussion by daisylb** [uxbox/uxbox#1477] [Ideas]
I've just been introduced to Penpot (thanks to [Andy Fitzsimon's Linux.conf.au talk](https://linux.conf.au/schedule/presentation/30/)), and I'm super excited to see something that has some of the features of tools like Figma that's open source! :O

**Are there any plans to add support for physical units (e.g. mm, cm, in) to Penpot?** I'd love to be able to use it to collaborate on branding for conferences I run, but I've yet to find anything (open source or not) that a) does really good live collaboration and b) supports physical units so that we can easily use it for things like posters, signage, badges, handouts, and the various bits of conference paraphernalia we need to print out, as well as screen stuff.

**Comment by diacritica** [comment]
Hi @excitedleigh ! I can't find the recording of that session. I'd be glad to pay the ticket if I know I'll be enjoying all the material even if it's not live. As for your question, I'll leave it to the rest of the team although my wild guess is that yes, we'll support physical units in the future. In the meantime, you can have your own "pixel to mm" conversion through the "export" scaling factor.

**Reply by slfhstr** [reply]
@diacritica are you meaning this for scaling ?
<img width="340" alt="Screenshot 2022-02-09 at 11 53 53" src="https://user-images.githubusercontent.com/28682881/153196073-edcfdcc2-9a1b-450b-97bd-b76fe2dd0c1a.png">

It seems to affect only relative size.  Not assigning pixels to mm.
How do we know what is correct relative size ?

I'm excited to use penpot as @excitedleigh mentions.
But I don't see how to do that.

**Reply by slfhstr** [reply]
Ah, I just saw that there is `size preset` with an option for A4 etc.
This is helpful !

**Reply by slfhstr** [reply]
It would still be helpful to have easy way to know pixels <--> mm
I need to create an A5 leaflet with a 3mm bleed area for edge-to-edge printing of final PDF.
I guess I use a calculator to derive this.  But would be nicer in the UI.

My guess is I need to do 794 pixels / 210 mm = 3.78px to 1mm
So bleed of 3mm each side = total 6mm to add to width and height of artboard = 22px (22.68 nut no option for decimal px and that's not logical)

I will experiment with this.

**Comment by slfhstr** [comment]
Seems to work quite well
Not exact
But maybe close enough
A4 width 816px ==> 210mm + 6mm total bleed should 216mm
A4 height 1145px ==> 297mm + 6mm total bleed should 303mm
<img width="321" alt="Screenshot 2022-02-09 at 12 10 40" src="https://user-images.githubusercontent.com/28682881/153198591-9d687174-e26f-4785-9942-5c4e06b48714.png">

Just in case this helps someone.

**Comment by NilsMoller** [comment]
4 years later, is this different today? Is there a feature request ticket we can look at for progress?
